app/gzac/build.gradle

```diff
@@ -19,7 +19,9 @@ dependencies {
     implementation(project(":objects-api"))
 
     // Database
-    implementation "mysql:mysql-connector-java"
+    //implementation "mysql:mysql-connector-java"
```

Comment:
How do we see this working in the future? We add both mysql and postgres?

Reply:
I think we just want to include both database drivers and have the spring configuration/profile decide which one gets used.